### ISSUE TITLE
Attaching debugger

### ISSUE THREAD

**Issue by DavidLozzi** [adobe/reactor-sync#9]
@bbythewa  Making great progress in figuring out how this works, documenting what files/folders are needed, etc. Is there a way to attach Code debugger to an instance so I can walk through the code? Doing a lot of console.logs now and it's getting old, quick ;) Thanks

**Comment by arobsco** [comment]
If you need any help I can pitch in! Spent quite a bit of time hacking this tool and other reactor packages to work with corporate proxies!

**Comment by DavidLozzi** [comment]
yes please @arobsco ! my big question now is how to actually attach a debugger.

**Comment by seLalit** [comment]
@DavidLozzi did you consider firing your script with '--inspect'? in my case I run the command in Debug mode using IDE Debug option, identify some pin points to add debugger and ideate the rest.
